ereuse_devicehub/resources/action/models.py

```diff
@@ -1727,6 +1727,12 @@ class MoveOnDocument(JoinedTableMixin, ActionWithMultipleTradeDocuments):
     container_to_id.comment = """This is the trade document used as container in a outgoing lot"""
 
 
+class Delete(ActionWithMultipleDevices):
```

Comment:
Name ambiguity between delete and deactivate a device. 

Actually the action deactivates a device, although to the user it may be seen as delete.

I propose to change the name of the class to Deactivate.